esi_leap/tests/api/controllers/test_types.py

```diff
@@ -11,16 +11,33 @@
 #    under the License.
 
 from esi_leap.api.controllers import types as types
+import mock as mock
 import unittest as unittest
+from wsme import types as wtypes
 
 
 class TestCollection(unittest.TestCase):
 
     def setUp(self):
         self.test_collection = types.Collection()
         self.test_collection._type = 'stuff'
-        self.test_collection.stuff = [1, 2, 3]
+        self.test_collection.stuff = []
+        self.obj1 = mock.Mock(uuid='aaaaa')
+        self.obj2 = mock.Mock(uuid='bbbbb')
+        self.obj3 = mock.Mock(uuid='ccccc')
+        self.test_collection.stuff.append(self.obj1)
+        self.test_collection.stuff.append(self.obj2)
+        self.test_collection.stuff.append(self.obj3)
```

Comment:
This can be simplified: self.test_collection.stuff = [self.obj1, self.obj2, self.obj3]